### PR TITLE
[fix](doc): incorrect tls config in java client doc

### PR DESCRIPTION
**Pull request by ericsyh**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

On the [java client loadconf](https://pulsar.apache.org/docs/next/client-libraries-java/#client) page there is a config `useTls` on the list but I think this config should be the `enableTls` and I also check on [ClientBuilder](https://pulsar.apache.org/api/client/org/apache/pulsar/client/api/ClientBuilder.html) there is no method as `useTls`

### Modifications

Replace the  incorrect tls config `useTls` by `enableTls` on the java client doc.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

**Comment by dave2wave** [comment]
I think that this looks correct, but there are conflicts to resolve

**Comment by ericsyh** [comment]
> I think that this looks correct, but there are conflicts to resolve

@michaeljmarshall @dave2wave Conflicts resolved. PTAL again.

**Comment by ericsyh** [comment]
cc @Anonymitaet Could help take a look at this Doc PR or assign a technical review?

File: site2/docs/client-libraries-java.md
Comment (Anonymitaet) [inline site2/docs/client-libraries-java.md:143]: what does `Deprecated` mean? `enableTls` is deprecated or something else？

File: site2/website/versioned_docs/version-2.1.0-incubating/client-libraries-java.md
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.1.0-incubating/client-libraries-java.md:478]: @momo-jun FYI: here is a missing occurrence (the `.md` issue)

**Review by Anonymitaet — COMMENTED** [review]
@gaoran10 
could you please review this PR from the technical perspective? Thank you!

File: site2/website/versioned_docs/version-2.1.0-incubating/client-libraries-java.md
Comment (momo-jun) [reply site2/website/versioned_docs/version-2.1.0-incubating/client-libraries-java.md:478]: Thanks for the reminder. I didn't have a chance to fix them in doc versions earlier than 2.10. There are hundreds of them in each version, so I need to implement it separately to avoid getting dizzy and crazy. Maybe I can file an good-first-issue :)

File: site2/docs/client-libraries-java.md
Comment (ericsyh) [reply site2/docs/client-libraries-java.md:143]: There is no need to config `enableTls` in pulsar client to enable Tls, just use `pulsar+ssl://` in the connection address

File: site2/docs/client-libraries-java.md
Comment (ericsyh) [reply site2/docs/client-libraries-java.md]: @nodece updated in https://github.com/apache/pulsar/pull/16315/commits/5d5d10252ec65fa0217d78747f62997d006dd3a3

Comment (nodece) [reply site2/docs/client-libraries-java.md]: Also you need to clean the `.enableTls(true)` in this documentation.

Comment (ericsyh) [reply site2/docs/client-libraries-java.md]: update in https://github.com/apache/pulsar/pull/16315/commits/f7d21fb776dcb90bd1a9ce4300f9385f090de094

File: site2/docs/client-libraries-java.md
Comment (Anonymitaet) [inline site2/docs/client-libraries-java.md:143]:
```suggestion
`enableTls` |boolean | Whether to use TLS encryption on the connection. Note that this parameter is **deprecated**. If you want to enable TLS, use `pulsar+ssl://` in `serviceUrl` instead. | false
```

do you mean this？

**Review by nodece — APPROVED** [review]

File: site2/docs/client-libraries-java.md
Comment (nodece) [reply site2/docs/client-libraries-java.md]: LGTM

File: site2/docs/client-libraries-java.md
Comment (nodece) [reply site2/docs/client-libraries-java.md:143]: You are right!

Comment (Anonymitaet) [reply site2/docs/client-libraries-java.md:143]: can you update it? @ericsyh

Comment (ericsyh) [reply site2/docs/client-libraries-java.md]: @Anonymitaet Updated in https://github.com/apache/pulsar/pull/16315/commits/9e233c5933a6c2caba4061270b62c40cc3632d76

**Review by tisonkun — APPROVED** [review]
Thanks for your contribution! LGTM.

@ericsyh may you tidy the PR description?

@Anonymitaet it seems this PR needs one more approval from committers to merge, could you help with reviewing?

**Comment by Anonymitaet** [comment]
@tisonkun thanks!

@ericsyh I've merged your PR with modifying the commit message to `[fix][doc] fix incorrect tls config in java client doc`

**Pull request merged**
